src/utils.rb

```diff
@@ -14,6 +14,16 @@ def commentify(string, comment_char: '#', line_length: 80)
           .join("\n")
       end
 
+      def run_command(command_class, *args, **options_hash)
+        options = Commander::Command::Options.new
```

Comment:
Now that we are running `Metalware` commands within our `metal` commands, we need to fix the logging. Currently we log what is in `ARGV` so running the second `Metalware` command will result in the first command being logged twice. This could be very confusing in the future.

Their are a few possible fixes:
1. Easy: Add a line of logging in `run_command` saying what the new command is (an except the old command gets logged twice)
1. Better: Store the old value of `ARGV` within `run_command` and switch it to what thew new command would be (even though it technically did get ran from the `cli`)
1. Best: Pass in a special tag into `BaseCommand` which will tell it not to log with `ARGV` but to construct a different log message based on it's inputs.

Reply:
> Now that we are running Metalware commands within our metal commands

This is not strictly true; we now run Metalware commands within the GUI, but the GUI itself is not a Metalware command. It's true that this is a bit of an issue though; if I check my recent logs when using the GUI I see various output lines like this:

```
I, [2017-09-07T17:08:45.407241 #7290]  INFO -- : metal 
```

I'm not sure which of your 'better' and 'best' suggestions is better; while it's true that the command didn't get run from the CLI it does still sound useful to have the log contain an accurate record of all Metalware commands run, however this was done.

Since the GUI is still a prototype I don't think this is key to fix right now, so I've logged this in https://trello.com/c/f6ktqiiM so that we come back to it.